dataactbroker/handlers/fileHandler.py

```diff
@@ -562,7 +567,8 @@ def checkGeneration(self, submission_id = None, file_type = None):
         if uploadJob.filename is None:
             responseDict["url"] = ""
         elif CONFIG_BROKER["use_aws"]:
-            responseDict["url"] = s3UrlHandler().getSignedUrl("d-files",uploadJob.filename, bucketRoute=None, method="GET")
+            path, file_name = uploadJob.filename.split("/")
```

Comment:
Would it make sense to future proof this a bit by allowing there to be multiple slashes in the filename?
```python
path, file_name = uploadJob.filename.rsplit("/", 1)
```